src/System.Runtime.Numerics/src/System/Numerics/BigNumber.cs

```diff
@@ -514,11 +502,18 @@ internal static char ParseFormatSpecifier(string format, out int digits)
             return (char)0; // Custom format
         }
 
-        private static string FormatBigIntegerToHexString(BigInteger value, char format, int digits, NumberFormatInfo info)
+        private static string FormatBigIntegerToHex(BigInteger value, char format, int digits, NumberFormatInfo info)
```

Comment:
It's still returning a string... any particular driver of the rename?

Reply:
a) Nothing else in the class that returns a String includes that in the name, e.g. FormatBigInteger rather than FormatBigIntegerString.
b) It'll probably change anyway for https://github.com/dotnet/corefx/issues/25336